cmd/edge-orchestration/capi/main.go

```diff
@@ -334,6 +337,7 @@ func SetHandler(iGetter C.identityGetterFunc, kGetter C.keyGetterFunc) {
 var count int
 var mtx sync.Mutex
 
+// PrintLog provides logging interface
 //export PrintLog
```

Comment:
How about removing this line?

Reply:
We got a [warnings](https://goreportcard.com/report/github.com/lf-edge/edge-home-orchestration-go#golint) from the Go Report Card